clinica/iotools/bids_utils.py

```diff
@@ -780,3 +780,33 @@ def run_dcm2niix(
 def write_to_tsv(dataframe: DataFrame, buffer: Union[PathLike, BinaryIO]) -> None:
     # Save dataframe as a BIDS-compliant TSV file.
     dataframe.to_csv(buffer, sep="\t", na_rep="n/a", date_format="%Y-%m-%d")
+
+
+def identify_modality(x: str) -> str:
+    """Identifies the modality of a file given its name.
+
+    Parameters
+    ----------
+    x: str
+        Input filename
+
+    Returns
+    -------
+    str:
+        Modality
```

Comment:
I think we should mention that the function returns `None` if the parsing isn't successful.